### ISSUE TITLE
HW4

### ISSUE THREAD

**Issue by jpatu001** [mikeizbicki/ucr-cs100#1711]
Hi, I typed in all my HW4 in my ucr-cs100 repo fork. Now I also had a few commits in it. How will I be able to submit a pull request without all that mutiple commits?  Since I have everything done, I want to submit it as just one whole commit. And btw I think my ucr-cs100 repo is behind of other commits as well, I hope that's not a problem.

**Comment by jpatu001** [comment]
I have tried the rebase guide. When I try gitpush origin patch-2 (this holds the squashed commit)
I get:
To https://github.com/jpatu001/ucr-cs100.git
 ! [rejected]        patch-2 -> patch-2 (non-fast-forward)
error: failed to push some refs to 'https://github.com/jpatu001/ucr-cs100.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

When I do git pull, and it gets reupdated to having seperate commits, I can git push. But the moment I have squashed them, I get the error above again.
